cfs_config_util/bin/main.py

```diff
@@ -48,14 +48,30 @@ def configure_logging():
     Returns:
         None.
     """
+    level = logging.DEBUG if verbose else logging.INFO
     console_log_format = '%(levelname)s: %(message)s'
     logger = logging.getLogger()
     console_handler = logging.StreamHandler()
-    console_handler.setLevel(logging.INFO)
+    console_handler.setLevel(level)
     console_formatter = logging.Formatter(console_log_format)
     console_handler.setFormatter(console_formatter)
     logger.addHandler(console_handler)
-    logger.setLevel(logging.INFO)
+    logger.setLevel(level)
+
+
+def log_args_and_exit(args):
+    from enum import Enum
+    import json
+
+    # Make the args.state value JSON serializable
+    class CustomEncoder(json.JSONEncoder):
+        def default(self, o):
+            if isinstance(o, Enum):
+                return o.value
+            return super().default(o)
+
+    LOGGER.debug(json.dumps(vars(args), cls=CustomEncoder, indent=4))
+    raise SystemExit(0)
```

Comment:
This is just for testing argument parsing. I will remove it when done testing that.